Guide/Constructive/Bonetrousle/README.md

```diff
@@ -62,6 +32,12 @@ formula above so that it does work. Note that the problem says that any combinat
 the needed number of noodles is a valid solution.
 </p>
```

Comment:
It allowed me to put comment on multiple lines without them appearing as different paragraphs.